functions/ts/kubeval/README.md

```diff
@@ -0,0 +1,36 @@
+# kubeval
+
+### Overview
+
+Validate KRM resources using [kubeval].
+
+### Synopsis
+
+`kubeval` allows you to validate KRM resources against their [json schemas].
+
+The json schemas can be provided via `schema_location` and
```

Comment:
The ordering of the content is still problematic. It's discussing a particular field out of nowhere:

https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/_template

First explain what the function config is? What kind is it? etc.

Reply:
If how usage docs should be written is not clear from the template, please expand the template. We don't want to have back and forth on every function.